open_flamingo/src/factory.py

```diff
@@ -14,6 +14,7 @@ def create_model_and_transforms(
     cross_attn_every_n_layers: int = 1,
     use_local_files: bool = False,
     decoder_layers_attr_name: str = None,
+    freeze_lm_embeddings: bool = False,
```

Comment:
maybe we should print a warning if we are doing this? I think it would reduce perf.